src/MAUI/Maui.Samples/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml.cs

```diff
@@ -50,11 +50,8 @@ public QueryFeatureCountAndExtent()
 
         private async Task Initialize()
         {
-            // Populate the ComboBox with states.
-            foreach (var state in _states)
-            {
-                StatesPicker.Items.Add(state.Key);
-            }
+            // Populate the Picker with states.
+            StatesPicker.ItemsSource = _states.Keys.ToList();
```

Comment:
I'm not sure what the technical limitation is for Maui but I think this also can just be set to _state.Keys since it's already a collection.

Reply:
It's unhappy converting a KeyCollection to Collections.IList for MAUI specifically - the ToList() gets around this